lib/jekyll/configuration.rb

```diff
@@ -175,7 +175,7 @@ def read_config_file(file)
       next_config
     rescue SystemCallError
       if @default_config_file ||= nil
-        Jekyll.logger.warn "Configuration file:", "none"
+        Jekyll.logger.warn "Configuration file:", "none (looking for '#{file}')"
```

Comment:
I think the path should be in a debug message (when you use `--verbose`). It's superfluous for most, I think.